examples/vision/handwriting_recognition.py

```diff
@@ -240,7 +264,7 @@ def distortion_free_resize(image, img_size):
         ],
     )
 
-    image = tf.transpose(image, perm=[1, 0, 2])
+    image = keras.ops.transpose(image, (1, 0, 2))
```

Comment:
You can just do `from keras import ops` then just use `ops.` instead of `keras.ops.` -- shorter

Reply:
Hi @fchollet - I am working on all the changes mentioned by you are update with new commit.